api_v3/services/MediaService.php

```diff
@@ -112,10 +112,10 @@ function addContentAction($entryId, KalturaResource $resource = null)
 
 		if ($resource)
 		{
-			$resource->validateEntry($dbEntry);
-			$kResource = $resource->toObject();
 			try
 			{
+				$resource->validateEntry($dbEntry, true);
```

Comment:
In case of exception EntryIdNotFound , there is no sense in canceling replace that is done inside handleErrorDuringSetResource please handle.